test/integration/targets/openssl_csr/tests/validate.yml

```diff
@@ -0,0 +1,17 @@
+- name: Validate CSR (test - privatekey modulus)
+  shell: 'openssl rsa -noout -modulus -in {{ output_dir }}/privatekey.pem | openssl md5'
+  register: privatekey_modulus
+
+- name: Validate CSR (test - Common Name)
```

Comment:
`-nameopt` switch should be used in order to be compatible with OpenSSL >= 1.1.0: the default format was [changed](https://github.com/openssl/openssl/commit/9cf315ef90d0cfeb8bb7a38873eea07a8f0dffec) between OpenSSL 1.0.2 and OpenSSL 1.1.0.

For example `openssl req -noout -subject -in csr -nameopt oneline,-space_eq` with `csr_cn.stdout.split('=')[-1]` could be used.